Sprint-3/implement/get-angle-type.js

```diff
@@ -29,13 +29,11 @@
 function getAngleType(angle){
     if (angle === 90) return "Right angle";
     else if (angle > 0 && angle < 90) return "Acute angle";
-    else if (angle > 90 && angle < 180) return "Obtuse angle";
+    else if (angle < 180) return "Obtuse angle";
```

Comment:
As the function will go through each conditional in order, line-by-line, typing in '0' will return 'obtuse angle'. How could this be remedied?